js/main.js

```diff
@@ -25,4 +30,69 @@ function handleNewSubmit(event) {
   var $newPhoto = document.querySelector('.new-entry-img');
   $newPhoto.setAttribute('src', 'images/placeholder-image-square.jpg');
   $newEntryForm.reset();
+  $entryList.prepend(generateEntryDOM(newEntry));
+  for (var i = 0; i < $viewList.length; i++) {
+    if ($viewList[i].getAttribute('data-view') === 'entries') $viewList[i].classList.remove('hidden');
+    else $viewList[i].classList.add('hidden');
+  }
+}
+
+function generateEntryDOM(entry) {
+// <li>
+//   <div class="row">
+//     <div class="column-half img-container">
+//       <img src="">
+//     </div>
+//     <div class="column-half">
+//       <h2></h2>
+//       <p></p>
+//     </div>
+//   </div>
+// </li>
+
+  var $newImage = document.createElement('img');
+  $newImage.setAttribute('src', entry.image);
+
+  var $imageBox = document.createElement('div');
+  $imageBox.className = 'column-half img-container';
+  $imageBox.appendChild($newImage);
+
+  var $newTitle = document.createElement('h2');
+  $newTitle.textContent = entry.title;
+
+  var $newDescription = document.createElement('p');
+  $newDescription.innerText = entry.notes;
+
+  var $textBox = document.createElement('div');
+  $textBox.className = 'column-half';
+  $textBox.appendChild($newTitle);
+  $textBox.appendChild($newDescription);
+
+  var $listContents = document.createElement('div');
+  $listContents.className = 'row';
+  $listContents.appendChild($imageBox);
+  $listContents.appendChild($textBox);
+
+  var $newListEntry = document.createElement('li');
+  $newListEntry.appendChild($listContents);
+
+  return $newListEntry;
+}
+
+function handleContentLoad(event) {
+  if (data.entries.length === 0) {
+    var $displayMessage = document.createElement('p');
+    $displayMessage.textContent = 'No entries have been recorded.';
+    $entryList.appendChild($displayMessage);
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    $entryList.appendChild(generateEntryDOM(data.entries[i]));
+  }
+}
+
+function newEntryClick(event) {
+  for (var i = 0; i < $viewList.length; i++) {
+    if ($viewList[i].getAttribute('data-view') === 'entry-form') $viewList[i].classList.remove('hidden');
+    else $viewList[i].classList.add('hidden');
```

Comment:
You have this same sort of behavior in the `handleNewPhotoURL` function above, this would be a good opportunity to create a function named something like `changeView` that will accept an argument of a string holding the name of the view you want to show. Then you can just reuse that function whenever you need to switch views